tests/integration/expected_metric_names.txt

```diff
@@ -9,7 +9,9 @@ k8s.cluster.nodes.ready
 k8s.cluster.nodes.ready.avg
 k8s.cluster.pods
 k8s.cluster.pods.running
+k8s.cluster.spec.cpu.requests
```

Comment:
These metrics were missing in the test data. They are here now because of the changes in the `utils/cleanup_mocked_prometheus_response.py` script.